### DISCUSSION TITLE
gaomon s620 tablet doesnt correctly display cursor + borderless crashes osu!lazer

### DISCUSSION THREAD

**Discussion by yallasm** [ppy/osu#16037] [Q&A]
help lol

**Comment by bdach** [comment]
![2021-12-11-003901_1691x308_scrot](https://user-images.githubusercontent.com/20418176/145654207-b028293d-5146-49da-b347-4117d9da4984.png)

did you perhaps miss this on the main discussions page?

if you did, here's that linked again: https://github.com/ppy/osu/discussions/15188. please go there, and read what it says, because there will be no help otherwise.

**Reply by frenzibyte** [reply]
In addition to the above, I recommend you read through https://github.com/ppy/osu/blob/1373d1cde271ef182bc98aa864c5ed6b959d38a6/CONTRIBUTING.md#i-would-like-to-submit-an-issue in detail (and issues = Q&A discussions here).
